pytalibstream/ema_keeper.py

```diff
@@ -0,0 +1,30 @@
+import pyslidingwindow
+
+
+class EmaKeeper:
+    def __init__(self, max_len: int):
+        self.arr = pyslidingwindow.SlidingWindowFloat(max_len)
+        self._ema = 0
+
+    def add(self, value: float):
+        if self.arr.length() < self.arr.max_len():
+            self.arr.push(value)
+            if self.arr.length() == self.arr.max_len():
+                self.ema = sum(self.arr)/self.arr.length()
+        else:
+            self.ema = self.get_ema(self.arr.max_len(), value, self.ema)
+
+        return self.ema
+
+    @property
+    def ema(self):
+        return self._ema
+
+    @ema.setter
+    def ema(self, new_ema):
+        self._ema = new_ema
```

Comment:
ideally, we do not want expose function variable to public. To avoid outside parts to set the sma by accident.
for example, we shouldn't allow external code to do this ```self.ema_keeper.ema = 3```
one solution is to not have this setting function, but directly use `self._ema = new_ema`